hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/OzoneRpcClientTests.java

```diff
@@ -1683,8 +1696,9 @@ private void bucketQuotaTestHelper(int keyLength, ReplicationConfig repConfig)
   }
 
   @ParameterizedTest
-  @MethodSource("bucketLayouts")
-  public void testBucketUsedNamespace(BucketLayout layout) throws IOException {
+  @MethodSource("bucketLayoutsWithEnablePaths")
+  public void testBucketUsedNamespace(BucketLayout layout, boolean enablePaths) throws IOException {
+    cluster.getConf().setBoolean(OmConfig.Keys.ENABLE_FILESYSTEM_PATHS, enablePaths);
```

Comment:
OM takes this setting from `OmConfig`.

```suggestion
    cluster.getOzoneManager().getConfig().setFileSystemPathEnabled(enablePaths);
```

(Use something like the original one when backporting.)

Probably would also make sense to restore the original setting in the `finally` block.